addons/project/models/project.py

```diff
@@ -1055,8 +1055,6 @@ def _get_default_stage_id(self):
 
     @api.model
     def _default_personal_stage_type_id(self):
-        if self._context.get('default_project_id'):
-            return False
```

Comment:
Why do we need this? It means that if I create a task for someone else, the default personal stage will be the one with the smallest sequence in mine? What was the need?

Reply:
@IT-Ideas  It was done to avoid searching on stage when there is default project on task, But here adding field in form view need default value of personal stage commit https://github.com/odoo/odoo/pull/86008/commits/a282fe9d22e50d69d662a3d6b4d2a6ccd313f49b.
It'll not set current user's personal stage if task user is not same as current one  [Line](https://github.com/odoo/odoo/pull/90588/files#diff-dcdc6e8a2e65aff0582c53d72c3890c1b56725a8b9fcf7abbb0378097b5999afR1063).